regulations/generator/sidebar/base.py

```diff
@@ -1,7 +1,5 @@
 import abc
```

Comment:
May need to bring back the __init__.py file for the sidebar directory. Some python things still expect it?